src/resources/js/routes/Users.svelte

```diff
@@ -14,6 +15,7 @@
   let users: User[] = [];
   let filteredUsers: User[] = [];
   let showModal = false;
+  let showUserDeleteModal = false;
```

Comment:
A shorter name like `showDeleteModal` might be better here.

Reply:
I was thinking that but wanted to be explicit for if we have like delte modal for service body or something

Reply:
The filename tells you what the context is. Since this is in `Users.svelte`, it's implied that you're deleting users.